needle/modules/configuration/mdm_assess.py

```diff
@@ -0,0 +1,131 @@
+from core.framework.module import BaseModule
+from core.device.device import Device
+from core.utils.menu import choose_from_list_data_protection, choose_from_list, choose_boolean
+from core.utils.utils import Utils
+import re, sys
+
+class Module(BaseModule):
+    meta = {
+        'name': 'MDM Assess',
+        'author': 'Oliver Simonnet (@MWRLabs)',
+        'description': 'Automated MDM Configuration Assessment tool.',
+        'options': (
+            ('template', True, True, 'Configuration template.[Plist|plutil-output]'),
+            ('verbosity', False, True, 'Output verbosity[1|2|3|4].')
+        ),
+        'comments': [
+            'VERBOSITY: 1 Displays misconfigurations only.',
+            'VERBOSITY: 2 Displays misconfiguration values.',
+            'VERBOSITY: 3 Displays recomendations.',
+            'VERBOSITY: 4 Displats all configutaion.']
+    }
+
+    # ==================================================================================================================
+    # UTILS
+    # ==================================================================================================================
+    def __init__(self, params):
+        BaseModule.__init__(self, params)
+        # Setting default output file
+        self.options['verbosity'] = 1
+
+    # Check if file is in Plist or a plutil-output format
+    def isPlist(self, configFile):
```

Comment:
This part of the code is not to check if the file is a genuine plist file (however that is also something that must be checked for error handling purposes) But to check if it is in its original plist XML format or  JSON format (plutil output)

I will alter comments for clarity.